### PR TITLE
Update demos to 0.2.1

### PR DESCRIPTION
**Pull request by denisvmedyantsev**

### CODE REVIEW

File: samples/README.md [hunk 1]
```diff
@@ -19,23 +19,23 @@ Sample Location: [peoplenet_detector](./peoplenet_detector)
...
+![](nvidia_car_classification/assets/nvidia-car-classification-loop-400.webp)
 
 ### Line crossing demo
 
 Sample Location: [line_crossing](./line_crossing)
 
 Preview:
 
-![](line-crossing-loop-400.webp)
+![](line_crossing/assets/line-crossing-loop-400.webp)
```
Comment (bwsw) [inline samples/README.md:38]: @denisvmedyantsev in the ticket, there is a note to rename it.

File: samples/line_crossing/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ The pipeline detects when people cross a user-configured line and the direction
 
 Preview:
 
-![](../line-crossing-loop.webp)
+![](assets/line-crossing-loop.webp)
 
 Tested on platforms:
```
Comment (bwsw) [inline samples/line_crossing/README.md:9]: @denisvmedyantsev 
pls, fix "graphana" to "grafana" in compose files. Oleg made a typo.

Comment (denisvmedyantsev) [reply samples/line_crossing/README.md:9]: fixed

File: samples/README.md (hunk 1 above)
Comment (denisvmedyantsev) [reply samples/README.md]: fixed

**Comment by bwsw** [comment]
@denisvmedyantsev 

image

File: samples/traffic_meter/README.md [hunk 3]
```diff
@@ -1,10 +1,10 @@
 # Line crossing demo
```
[the remark this answers is not in the document]
Comment (denisvmedyantsev) [reply samples/traffic_meter/README.md]: renamed

**Pull request merged**